db/system_keyspace.cc

```diff
@@ -52,6 +52,21 @@
 
 using days = std::chrono::duration<int, std::ratio<24 * 3600>>;
 
+template <>
+inline sstables::basic_info&& value_cast<sstables::basic_info>(data_value&& value) {
```

Comment:
Where is this method used?

Reply:
It is indirectly used here https://github.com/scylladb/scylladb/pull/21288/files#diff-5bcbb172f03dae93b568d237d4761d4b1bb01ae8ea23c3ed4279153aa18a5f61R2482.